contracts/reward/src/entry.rs

```diff
@@ -298,21 +314,38 @@ pub fn main() -> Result<(), Error> {
     Ok(())
 }
 
-fn calculate_reward(miner: &Vec<u8>, epoch_reward_obj: &EpochRewardObject) -> Result<u128, Error> {
+fn get_minimum_normal_propose_count(metadata: &MetadataCellData) -> u64 {
+    let metadata = metadata.metadata().get(0);
+    let epoch_total_block_num = metadata.epoch_len() * metadata.period_len();
+    let validator_num = metadata.validators().len();
+    (epoch_total_block_num * 95 / validator_num as u32 / 100).into()
```

Comment:
Will 95 not be changed?

Reply:
Does `95` need to be moved to metadata cell?

Reply:
let it be fixed, may change it in the future